src/pages/documentation/adobe-journey-optimizer/release-notes.md

```diff
@@ -9,6 +9,18 @@ keywords:
 
 # Release Notes
 
+## September 28, 2023
+
+### iOS Messaging 4.1.0
+- Added new notifcation tracking API `handleNotificationResponse` with parameters UNNotificationResponse and an optional callback the returns `PushTrackingStatus` enum.
+- The new API will automatically handle the click behaviour (OPENAPP, WEBURL, DEEPLINK) defined for the notification.
```

Comment:
I think that, since this is part of the above point, making this a sub-bullet of the previous one would be better.